src/avt/Blueprint/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#*****************************************************************************
+#
+# Copyright (c) 2000 - 2022, Lawrence Livermore National Security, LLC
```

Comment:
Please use shortened form of copyright notice. See root CMakeLists.txt for example.